spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -121,18 +121,30 @@ def verify_session_cookie
     describe 'new' do
       context 'routes not requiring auth' do
         %w[mhv dslogon idme].each do |type|
-          it "routes /sessions/#{type}/new to SessionsController#new with type: #{type}" do
+          it "routes /sessions/#{type}/new to SessionsController#new with type: #{type} and returns JSON" do
+            request.headers[:Accept] = 'application/json'
```

Comment:
Instead of ` request.headers[:Accept] = 'application/json'` how about `get(:new, type: type, format: :json)`?

Reply:
That does look better. I'll make that change.